__docs__/wonder-blocks-form/checkbox.stories.tsx

```diff
@@ -96,6 +105,77 @@ Indeterminate.parameters = {
     },
 };
 
+export const IndeterminateWithGroup: StoryComponentType = () => {
+    const [allSelected, setAllSelected] = React.useState<boolean | null>(false);
+    const [selectedValues, setSelectedValues] = React.useState<Array<string>>(
+        [],
+    );
+    const choices = [
+        {label: "Pepperoni", value: "pepperoni"},
+        {label: "Sausage", value: "sausage"},
+        {label: "Extra cheese", value: "cheese"},
+        {label: "Green pepper", value: "pepper"},
+        {label: "Mushroom", value: "mushroom"},
+    ];
+
+    const handleSelectAll = () => {
+        if (allSelected || allSelected === null) {
+            setSelectedValues([]);
+            setAllSelected(false);
+        } else {
+            const allValues = choices.map((choice) => choice.value);
+            setSelectedValues(allValues);
+            setAllSelected(true);
+        }
+    };
+
+    const handleCheckboxGroupSelect = (values: Array<string>) => {
+        setSelectedValues(values);
+        if (values.length === choices.length) {
+            setAllSelected(true);
+        } else if (values.length) {
+            setAllSelected(null);
+        } else {
+            setAllSelected(false);
+        }
+    };
+
+    return (
+        <View>
+            <Checkbox
+                checked={allSelected}
+                label={"Topping(s)"}
+                onChange={handleSelectAll}
+            />
+            <Strut size={Spacing.small_12} />
+            <View style={{marginInlineStart: Spacing.large_24}}>
+                <CheckboxGroup
+                    groupName="toppings"
+                    onChange={handleCheckboxGroupSelect}
+                    selectedValues={selectedValues}
+                >
+                    {choices.map((choice) => (
+                        <Choice
+                            key={choice.label}
+                            label={choice.label}
+                            value={choice.value}
+                        />
+                    ))}
+                </CheckboxGroup>
+            </View>
+        </View>
+    );
+};
+
+IndeterminateWithGroup.parameters = {
+    docs: {
+        description: `Here is an example of how you can use the
```

Comment:
Same comment here about adding the `story` key.

https://5e1bf4b385e3fb0020b7073c-nwtpfilbpc.chromatic.com/?path=/docs/form-checkbox--docs#indeterminate-with-group
<img width="1085" alt="Screenshot 2023-08-01 at 10 39 16 AM" src="https://github.com/Khan/wonder-blocks/assets/843075/4fae00a3-8bcd-4c97-a27a-f5b304a28b32">